### ISSUE TITLE
Simulators 1.5.12 Text displays without function

### ISSUE THREAD

**Issue by yak55** [FrSkyRC/ETHOS-Feedback-Community#4270]
Where is the correct storage location for documents/users?

The simulator can no longer find the texts.

Thank you

Greetings Eugen

**Comment by philippe-warnotte** [comment]
X20S and V20 Simulator 1.5.11 and 1.5.12.
I can’t create or modify Widget Bitmap file !
But I can see Bitmaps in file manager and edit model. NOT in widget edit.
Idem for texts
![image](https://github.com/user-attachments/assets/251d4bfa-63ba-4f5f-8540-61c5442fee9a)

![image](https://github.com/user-attachments/assets/d16211b9-1379-46ed-a7cc-6f816c2c3b51)
![image](https://github.com/user-attachments/assets/f2159330-1cb3-4fc3-bb49-f91ee6802933)

**Comment by bsongis-frsky** [comment]
After your message, I will change a little bit the behaviour with 1.5.12
1) if audio / documents / scripts exist in the user directory, then it will be used in priority
2) if it doesn't exist, then the system directory will be used (C:\Program Files\...)
Would you please check it in the next nightlies? Thanks in advance!

**Comment by philippe-warnotte** [comment]
Hello, I installed your last version X20S simulator 1.5.12 Nightly.  I have copied my Bitmaps perso into :
![image](https://github.com/user-attachments/assets/84e184c1-ae17-41b2-b061-7ce27bd02bec)

 
And copy also into : user path
![image](https://github.com/user-attachments/assets/ac1685c7-de9a-4f8f-83c7-1c4c668b948d)

 
Create and Edit new model, find my BMP OK
![image](https://github.com/user-attachments/assets/ce1cd609-e476-4e38-80e0-fbcb5490d8a2)

 
EDIT WIDGET : NO find path Bitmap File list ! 
![image](https://github.com/user-attachments/assets/90419479-5e05-464a-b614-4768e8d998c3)

 
For Simulator V20 it’s the same issue. 
V20, no user path created during installation. But that's not the issue...
I try that on 2 differents PC Windows 10.
